app/views/transitive_dependencies/_dependencies.html.erb

```diff
@@ -0,0 +1,22 @@
+<% if dependencies.present? && @latest_version.indexed %>
+  <div class="dep-full--l">
+    <% dependencies.each do |dependency| %>
+      <% if rubygem = dependency.rubygem %>
+        <% if @gem_deps_names.include?(rubygem.name) %>
+          <div><span class="deps_item deps_item--duplicate"><%= rubygem.name %> <%= dependency.clean_requirements %></span></div>
```

Comment:
we need "transitive_dependencies/dependencies" because I am rendering it with different dependencies during the ajax requests(loading new dependencies)
https://github.com/rubygems/rubygems.org/pull/2029/commits/3163ce1c2ce021faba96cb44f41761f4fb27bd1b#diff-dd903fc5354c928ae886bbf9706832deR4

I actually added the link to the gem in the list of dependencies on the right hand side. I can add it here too.

Reply:
dependencies list on the left side feels redundant. Is the `transitive_dependencies/dependencies` providing any additional info to the user which doesn't already exist on the page?

Reply:
Do you mean the list on the right side(dependency list)? https://user-images.githubusercontent.com/16337625/60026770-d77e3700-96b9-11e9-8f39-4f6a1cc39468.png

Is it redundant? I can remove it if so. I just added it for users to get a list in alphabetical order.